jsapp/js/components/account/mfaModals.tsx

```diff
@@ -0,0 +1,606 @@
+import React from 'react'
+import bem, {makeBem} from 'js/bem'
+import { stores } from 'jsapp/js/stores'
+import QRCode from 'qrcode.react'
+import Button from 'js/components/common/button'
+import TextBox from 'js/components/common/textBox'
+import mfaActions, {
+  MfaActivatedResponse,
+  MfaBackupCodesResponse,
+} from 'js/actions/mfaActions'
+
+import './mfaModals.scss'
+
+bem.MFAModals = makeBem(null, 'mfa-modal')
+bem.MFAModals__qrstep = makeBem(bem.MFAModals, 'qrstep')
+bem.MFAModals__backupstep = makeBem(bem.MFAModals, 'backupstep')
+bem.MFAModals__manualstep = makeBem(bem.MFAModals, 'manualstep')
+bem.MFAModals__tokenstep = makeBem(bem.MFAModals, 'tokenstep')
+bem.MFAModals__disclaimerstep = makeBem(bem.MFAModals, 'disclaimerstep')
+bem.MFAModals__helptextstep = makeBem(bem.MFAModals, 'help-text-step')
```

Comment:
I would say all these `…step` are not very BEM valid. We should use `bem.MFAModals` with modifiers - e.g. `<bem.MFAModals m='step-qr'>` etc.